<?xml version="1.0"?>
<launch>
  <!-- Arguments -->
  <arg name="initial_motor_power" default="on"/>

  <!-- Nodes -->
  <node pkg="raspicat_gazebo" type="motors_sim" name="motors_sim" required="true" output="screen" 
        args="$(arg initial_motor_power)">
    <param name="check_cmdvel_rate"     value="10"/>
    <param name="num_callback_threads"  value="1"/>
  </node>

  <node pkg="raspicat_gazebo" type="ultrasonic_convert" name="ultrasonic_convert" required="true" output="screen">
    <param name="publish_rate"          value="10"/>
    <param name="num_callback_threads"  value="1"/>
  </node>
</launch>